src/Stripe.net/Infrastructure/Public/SystemNetHttpClient.cs

```diff
@@ -204,29 +214,11 @@ private string BuildStripeClientUserAgentString()
                 { "bindings_version", StripeConfiguration.StripeNetVersion },
                 { "lang", ".net" },
                 { "publisher", "stripe" },
-                { "lang_version", RuntimeInformation.GetLanguageVersion() },
+                { "lang_version", RuntimeInformation.GetRuntimeVersion() },
```

Comment:
Is using the same property the right decision here? Won't it skew our results and make it harder to inspect?

Reply:
The current value is more or less useless (it tells us the proportion of Framework vs. Core users, but no useful version numbers). I think we can start sending meaningful values and update our internal metrics to only look at the `lang_version` field when the bindings version is >= 35.0.